var/spack/repos/builtin/packages/openldap/package.py

```diff
@@ -31,62 +31,69 @@ class Openldap(AutotoolsPackage):
             values=('gnutls', 'openssl'), multi=False)
 
     variant('perl', default=False, description='Perl backend to Slapd')
+    variant('sasl', default=True,  description='Build with Cyrus SASL support')
+    variant('static', default=False, description='Build static libraries')
+    variant('shared', default=True, description='Build shared libraries')
+    variant('dynamic', default=True, description='Enable linking built binaries with dynamic libs')
+    conflicts('~static', when='~shared')
 
     depends_on('icu4c', when='+icu')
     depends_on('gnutls', when='~client_only tls=gnutls')
     depends_on('openssl', when='~client_only tls=openssl')
+    depends_on('openssl@1.1.1:', when='~client_only tls=openssl @2.6.0:')
     depends_on('unixodbc', when='~client_only')
     depends_on('postgresql', when='~client_only')
     depends_on('berkeley-db', when='~client_only')  # for slapd
     # Recommended dependencies by Linux From Scratch
-    # depends_on('cyrus-sasl', when='~client_only') # not avail. in spack yet
+    depends_on('cyrus-sasl', when='+sasl')
     # depends_on('openslp', when='~client_only') # not avail. in spack yet
     # depends_on('Pth', when='~client_only') # not avail. in spack yet
     depends_on('perl', when='~client_only+perl')  # for slapd
     depends_on('groff', type='build')
+    depends_on('pkgconfig', type='build')
+    depends_on('wiredtiger', when='@2.6.0:')
```

Comment:
It looks like this is optional with `--enable-wt` and also it looks like it might be off by default. If it is optional, can you add a variant for it?

Reply:
It should be optional, but on my test machine (Ubuntu 20.04) it tries to find WT anyway:
```
     330    checking for WT... no
  >> 331    configure: error: Package requirements (wiredtiger) were not met:
     332
     333    Package 'wiredtiger', required by 'virtual:world', not found
     334
     335    Consider adjusting the PKG_CONFIG_PATH environment variable if you
     336    installed software in a non-standard prefix.
     337
```

I can disable it explicitly since I don't build openldap server